components/eam/src/physics/crm/physpkg.F90

```diff
@@ -800,24 +803,30 @@ subroutine phys_run1(phys_state, ztodt, phys_tend, pbuf2d,  cam_in, cam_out)
   !-----------------------------------------------------------------------------
   ! CRM physics
   !-----------------------------------------------------------------------------  
+
+  ! Determine column start and end indices for crm_ecpp_output
+  ncol_sum = 0
   do c=begchunk, endchunk
+    ncol_beg(c) = ncol_sum + 1
```

Comment:
Probably makes more sense to call these icol_beg and icol_end since they are indices to the start and end of the chunk? I would also maybe argue that this block belongs closer to where it’s used, since it only appears to be used in one place right before the call to tphysbc2.

Reply:
ok, I'll do that.